azure-functions/create-function-app-connect-to-document-db/create-function-app-connect-to-documentdb.sh

```diff
@@ -7,14 +7,8 @@ az storage account create --name myfunctionappstorage --location westeurope --re
 # create a new function app, assign it to the resource group you have just created
 az functionapp create --name myFunction --resource-group myResourceGroup --storage-account myfunctionappstorage --consumption-plan-location westeurope
 
-# create documentdb database. 
+# create DocumentDB database. 
 az documentdb create --name myDocumentDB --resource-group myResourceGroup
 
-
-
-
-
-
-
-
-
+# configure function app settings to use DocumentDB connection string
+az functionapp config appsettings update --name MyFunction --resource-group myResourceGroup --setting myDocumentDBConnectionString="AccountEndpoint=https://mydocumentdb.documents.azure.com:443/;AccountKey=MyKey;"
```

Comment:
We should get the actual connection details as they are achievable through script. Here is a sample for DocDB: https://github.com/Azure/azure-docs-cli-python-samples/blob/master/app-service/connect-to-documentdb/connect-to-documentdb.sh